vfs/vfscache/item_test.go

```diff
@@ -173,7 +173,7 @@ func TestItemTruncateExisting(t *testing.T) {
 	require.Error(t, item.Truncate(40))
 	checkObject(t, r, "existing", contents)
 
-	require.NoError(t, item.Open(obj))
```

Comment:
I'm not sure the purpose of this change - I would have thought the test should work without it.

Reply:
As discussed in our PMs, this is due to some incorrect revert of some lines of an earlier commit.  Will fix.